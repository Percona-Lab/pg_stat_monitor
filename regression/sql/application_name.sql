CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();

SELECT 1 AS num;
SELECT query,application_name FROM pg_stat_monitor ORDER BY query COLLATE "C";
SELECT pg_stat_monitor_reset();

SELECT 1 AS num;
SET pg_stat_monitor.pgsm_track_application_names='no';
SELECT 1 AS num;
SELECT query,application_name FROM pg_stat_monitor ORDER BY query, application_name COLLATE "C";
SELECT pg_stat_monitor_reset();

DROP EXTENSION pg_stat_monitor;
