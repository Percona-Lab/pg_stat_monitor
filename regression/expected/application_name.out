CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

SELECT 1 AS num;
 num 
-----
   1
(1 row)

SELECT query,application_name FROM pg_stat_monitor ORDER BY query COLLATE "C";
             query              |      application_name       
--------------------------------+-----------------------------
 SELECT 1 AS num                | pg_regress/application_name
 SELECT pg_stat_monitor_reset() | pg_regress/application_name
(2 rows)

SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

SELECT 1 AS num;
 num 
-----
   1
(1 row)

SET pg_stat_monitor.pgsm_track_application_names='no';
SELECT 1 AS num;
 num 
-----
   1
(1 row)

SELECT query,application_name FROM pg_stat_monitor ORDER BY query, application_name COLLATE "C";
                         query                         |      application_name       
-------------------------------------------------------+-----------------------------
 SELECT 1 AS num                                       | pg_regress/application_name
 SELECT 1 AS num                                       | 
 SELECT pg_stat_monitor_reset()                        | pg_regress/application_name
 SET pg_stat_monitor.pgsm_track_application_names='no' | 
(4 rows)

SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

DROP EXTENSION pg_stat_monitor;
